cfme/utils/appliance/implementations/ui.py

```diff
@@ -214,13 +214,14 @@ def before_click(self, element, locator):
         # this is necessary in order to handle unexpected alerts like "Abandon Changes"
         self.browser.page_dirty = self.page_has_changes
 
-    def after_click(self, element, locator):
+    def after_click(self, element, locator, ignore_ajax=None):
```

Comment:
We must be careful about this, because the class method we're overriding does not take additional kwargs.

Reply:
I updated class method in wt.core as well

Reply:
I see your REQUIRES line now! :bowing_man: